dask/array/tests/test_slicing.py

```diff
@@ -250,32 +250,37 @@ def test_slicing_with_newaxis():
 def test_take():
     chunks, dsk = take('y', 'x', [(20, 20, 20, 20)], [5, 1, 47, 3], axis=0)
     expected = {('y', 0): (getitem, (np.concatenate,
-                                     [(getitem, ('x', 0), ([1, 3, 5],)),
-                                      (getitem, ('x', 2), ([7],))], 0),
-                           ([2, 0, 3, 1], ))}
-    assert dsk == expected
+                                     [(getitem, ('x', 0), (np.array([1, 3, 5]),)),
+                                      (getitem, ('x', 2), (np.array([7]),))], 0),
+                           (np.array([2, 0, 3, 1]), ))}
+    np.testing.assert_equal(dsk.values(), expected.values())
+    assert list(dsk.keys()) == list(expected.keys())
```

Comment:
Instead of those two lines, would the following work?
```
np.testing.assert_equal(sorted(dsk.items()), sorted(expected.items()))
```
(I'm using sorted() since dicts are unordered)